manticore/platforms/evm.py

```diff
@@ -1049,6 +1051,41 @@ def __reduce__(self):
         return (self.__class__, (self.data, ))
 
 
+def concretized_args(**policies):
+    '''
+    Make sure an EVM instruction has all of its arguments concretized according to
+    provided policies.
+
+    Example decoration:
+
+        @concretized_args(size='ONE')
+        def LOG(self, address, size, *topics):
+            ...
+
+    The above will make sure that the |size| parameter to LOG is Concretized when symbolic
+    according to the 'ONE' policy.
+
+    :param policies: A kwargs list of argument names and their respective policies.
+                         Provide None or '' as policy to use default.
+    :return: A function decorator
+    '''
+    def concretizer(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            spec = inspect.getargspec(func)
+            sys.stderr.write("args: {}, kwargs: {}".format(repr(args), repr(kwargs)))
+            for arg, policy in policies.items():
+                assert arg in spec.args, "Concretizer argument not found in wrapped function."
+                policy = policy or 'MINMAX'
+                # index is 0-indexed, but ConcretizeStack is 1-indexed. However, this is correct
+                # since implementation methods always take a first `self` parameter.
```

Comment:
Its called `bound method` because the `self` is bound ;D
```
In [11]: class Cls:
    ...:     def a(self):
    ...:         print('a')
    ...:         
    ...:     

In [12]: Cls.a, Cls().a
Out[12]: 
(<function __main__.Cls.a>,
 <bound method Cls.a of <__main__.Cls object at 0x7fabe5ef5a90>>)

In [13]: Cls.a(Cls())
a

In [14]: Cls().a()
a

```